_layouts/activity-default.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="{{ page.lang }}">
+
+  {% include head.html %}
+
+  <body>
+
+    {% include header.html %}
+
+    <div class="page-content">
+      <div class="wrapper activity-page">
```

Comment:
Been there, tried stuff. Jekyll has no page information at that level. Couldn't find a better way.